extensions/clickup/src/types/task.dt.ts

```diff
@@ -22,17 +22,17 @@ export interface Task {
   points: null;
   time_estimate: null;
   time_spent: number;
-  custom_fields: any[];
-  dependencies: any[];
-  linked_tasks: any[];
+  custom_fields?: CustomField;
+  // dependencies: any[];
```

Comment:
Commented out code?

Reply:
I couldn't find proper references for these types in the API docs. But I parsed it from API responses manually right now. Also resolved ✅.